app/scripts/views/settings/avatar_crop.js

```diff
@@ -103,6 +108,26 @@ function (p, Cocktail, FormView, SettingsMixin, AvatarMixin, Template,
           self.navigate('settings');
           return result;
         });
+    },
+
+    _onRotate: function () {
+      this.logScreenEvent('rotate.cw');
```

Comment:
These events need to be added [to the docs](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md)